Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/EditorContextMenu.cpp

```diff
@@ -20,25 +19,19 @@ namespace AzToolsFramework
         if (mouseInteraction.m_mouseInteraction.m_mouseButtons.Right() &&
             mouseInteraction.m_mouseEvent == ViewportInteraction::MouseEvent::Down)
         {
-            contextMenu.m_shouldOpen = true;
             contextMenu.m_clickPoint =
                 ViewportInteraction::QPointFromScreenPoint(mouseInteraction.m_mouseInteraction.m_mousePick.m_screenCoordinates);
         }
 
-        // disable shouldOpen if right clicking an moving the mouse
-        if (mouseInteraction.m_mouseEvent == ViewportInteraction::MouseEvent::Move)
-        {
-            const QPoint currentScreenCoords =
-                ViewportInteraction::QPointFromScreenPoint(mouseInteraction.m_mouseInteraction.m_mousePick.m_screenCoordinates);
-
-            contextMenu.m_shouldOpen = contextMenu.m_shouldOpen && (currentScreenCoords - contextMenu.m_clickPoint).manhattanLength() < 2;
-        }
-
         // do show the context menu
         if (mouseInteraction.m_mouseInteraction.m_mouseButtons.Right() &&
             mouseInteraction.m_mouseEvent == ViewportInteraction::MouseEvent::Up)
         {
-            if (contextMenu.m_shouldOpen)
+            const QPoint currentScreenCoords =
+                ViewportInteraction::QPointFromScreenPoint(mouseInteraction.m_mouseInteraction.m_mousePick.m_screenCoordinates);
+
+            // if the mouse hasn't moved, open the pop-up menu
+            if ((currentScreenCoords - contextMenu.m_clickPoint).manhattanLength() < 2)
```

Comment:
nitpick - I'd suggest a named constant here rather than 2. And might want to expose it at some point to improve accessibility for people with motor problems.

Reply:
That's true - this is old and was there before but I can totally move to an AZ_CVAR for now which will help in future it it comes up as an issue